src/features/earn/providers/yearn/constants/vaults.ts

```diff
@@ -36,7 +36,7 @@ export const SUPPORTED_VAULTS: SupportedYearnVault[] = [
     type: EarnType.Vault
   },
   {
-    vaultAddress: toLower('0x5f18C75AbDAe578b483E5F43f12a39cF75b973a9'),
+    vaultAddress: toLower('0xa354f35829ae975e850e23e9615b11da1b3dc4de'),
```

Comment:
see comment, duplicate this vault, name the original `USDC 0.3.0` and the new one `USDC 0.4.3` until we have a migration pathway